packages/devtools_app/lib/src/screens/network/network_controller.dart

```diff
@@ -19,6 +19,31 @@ import 'network_model.dart';
 import 'network_screen.dart';
 import 'network_service.dart';
 
+/// Different types of Network Response which can be used to visualise response
+/// on Response tab
+enum NetworkResponseType {
```

Comment:
nit: rename to NetworkResponseViewType for consistency and to differentiate between different "types" and "view types" for responses